readthedocs_ext/readthedocs.py

```diff
@@ -22,7 +21,62 @@
 ]
 
 
+def finalize_media(app):
+    """
+    Point media files at our media server.
+    """
```

Comment:
Would be easy to enable on this repo at least, since it's tiny. I can work on that.